src/pages/folder/FolderPage.js

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getUserFolders, getUserLinks } from '../../api/folder';
+import './folderPage.css';
+import AddLinkInput from './components/addLinkInput/AddLinkInput';
+import SearchBar from '../../components/searchBar/SearchBar';
+import Card from '../../components/card/Card';
+import EmptyPage from './components/emptyPage/EmptyPage';
+import OptionButton from './components/optionButton/OptionButton';
+import FloatingButton from '../../components/floatingButton/FloatingButton';
+import OPTION_ICONS from './constant';
+
+import addIcon from '../../assets/folder/add.svg';
+import addPrimaryIcon from '../../assets/folder/addPrimaryColor.svg';
+import SortButton from './components/sortButton/SortButton';
+
+export default function FolderPage() {
+  const [links, setLinks] = useState([]);
+  const [folders, setFolders] = useState([]);
+  const [searchParams, setSearchParams] = useSearchParams();
+  const initFolderId = searchParams.get('folderId');
+  const [folderId, setFolderId] = useState(initFolderId || null);
+  // eslint-disable-next-line no-unused-vars
+  const [isClicked, setIsClicked] = useState(false);
+  const [categoryTitle, setCategoryTitle] = useState('전체');
+  const [viewPortWidth, setViewPortWidth] = useState(window.innerWidth);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setViewPortWidth(window.innerWidth);
+    };
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  const isMobileView = viewPortWidth <= 767;
+
+  const fetchUserFolders = async () => {
+    const result = await getUserFolders();
+    const { data } = result;
+    setFolders(data);
+  };
+
+  const fetchUserLinks = async (id) => {
+    const result = await getUserLinks(id);
+    const { data } = result;
+    setLinks(data);
+  };
+
+  useEffect(() => {
+    fetchUserFolders();
+  }, []);
+
+  useEffect(() => {
```

Comment:
ㅎㅎㅎ 굳굳! 다만 folderId가 없는 경우는 실행하지않아야겠죠?
if문으로 예외처리해주세요!

```suggestion
  useEffect(() => {
    if(!folderId) return;
```